src/sources/http_server.rs

```diff
@@ -484,6 +485,16 @@ impl HttpSource for SimpleHttpSource {
                         SimpleHttpConfig::NAME,
                         now,
                     );
+
+                    source_ip.map(|addr| {
+                        self.log_namespace.insert_source_metadata(
+                            SimpleHttpConfig::NAME,
+                            log,
+                            Some(LegacyKey::Overwrite(path!("host"))),
+                            path!("host"),
+                            addr.to_string(),
```

Comment:
I think this necessitates an update to the `schema_definition()` function.

Some background context in https://github.com/vectordotdev/vector/blob/master/docs/tutorials/lognamespacing.md

This should also (hopefully) prompt a unit test change as well.

Reply:
Hey @neuronull, I had one more concern regarding using `"host"` instead of `host_key`. Should, I use host key as they key for this instead of "host"?

Reply:
Ah, I believe "host" is the correct usage, as you have here.